service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceDeleteFlight.java

```diff
@@ -25,22 +26,30 @@ public WorkspaceDeleteFlight(FlightMap inputParameters, Object applicationContex
     WorkspaceStage workspaceStage =
         WorkspaceStage.valueOf(
             inputParameters.get(WorkspaceFlightMapKeys.WORKSPACE_STAGE, String.class));
+    UUID workspaceId =
+        UUID.fromString(inputParameters.get(WorkspaceFlightMapKeys.WORKSPACE_ID, String.class));
     // TODO: we still need the following steps once their features are supported:
```

Comment:
I don't know if we need to comment here about these systems that don't exist, though I suppose it's a logical place for it. It's not really a TODO because too many things have to happen first.

Reply:
Yeah - I found the old comment useful for context when I was making this change so I'm inclined to leave it, but it's not really something we can do yet